airlock/views/workspace.py

```diff
@@ -15,6 +16,9 @@
 from .helpers import get_path_item_from_tree_or_404, get_workspace_or_raise, serve_file
 
 
+tracer = trace.get_tracer_provider().get_tracer(__name__)
```

Comment:
Yeah, thinking about tracers ~= logs is a reasonable default.

It basically boils down to an attribute we can filter on with the span data.

So what would be a useful filter? I am not really sure here.  But `__name__` is probably not that useful?

In jobrunner, we have 3 explicit tracers - jobs, ticks, and loops - for 3 different traces of activity.  Here, we only really have one activity, pattern, HTTP requests.  You could make an argument for this being the "airlock" tracer, perhaps?

Anyway, this is just a comment to say, I'm not 100% clear on best practice for naming your tracer objects :shrug:

Reply:
I basically just copied it from somewhere in job-server; originally I did just call it airlock. Maybe that's better, then